easycv/datasets/loader/sampler.py

```diff
@@ -464,3 +465,73 @@ def __len__(self):
 
     def set_epoch(self, epoch):
         pass
+
+
+class RASampler(torch.utils.data.Sampler):
+    """Sampler that restricts data loading to a subset of the dataset for distributed,
```

Comment:
I don't quite understand what the `RASampler` is used for. For which models?  which scenarios?

Reply:
RASampler is used for sampling of DeiT III.
RASampler (Repeated Augmentation Sampler): Assuming the batch size is n, the n pictures selected from the dataset will be copied to m copies, that is, a total of m*n pictures. Then, the n pictures will be randomly selected from the m*n pictures to obtain the final sampling result. Therefore, there may be duplicate pictures in the final n pictures, which may become different after data augment. So RASampler can enable the model to learn the augment-invariant features. The authors of DeiT III believe that RASampler may be helpful.